package.json

```diff
@@ -14,20 +14,37 @@
   },
   "license": "MIT",
   "dependencies": {
-    "hoek": "2.x.x"
-  },
-  "peerDependencies": {
-    "hapi": ">= 8.x.x < 10"
+    "autoprefixer-stylus": "0.8.0",
+    "axis": "0.5.x",
+    "clean-webpack-plugin": "0.1.3",
+    "compression-webpack-plugin": "0.2.0",
+    "css-loader": "0.19.0",
+    "hoek": "2.x.x",
+    "html-loader": "0.3.0",
+    "html-webpack-plugin": "^1.6.2",
+    "jade": "1.11.0",
+    "jeet": "6.1.x",
+    "rupture": "0.6.1",
+    "style-loader": "0.12.x",
+    "stylus-loader": "1.3.x",
+    "webpack": "1.12.2"
   },
   "devDependencies": {
+    "belly-button": "1.x.x",
     "code": "1.x.x",
     "hapi": "9.x.x",
+    "inert": "3.x.x",
     "joi": "6.x.x",
-    "lab": "6.x.x",
-    "semistandard": "7.x.x"
+    "lab": "5.x.x",
+    "vision": "3.x.x"
+  },
+  "peerDependencies": {
+    "hapi": "9.x.x"
   },
   "scripts": {
-    "test": "semistandard && lab -v -t 100 -a code"
+    "test": "belly-button -i 'lib/*.js' -i dev-server.js -i webpack.config.js && lab -v -t 100 -a code",
```

Comment:
Hm, I guess belly-button is already in here. When belly-button requires some configuration, I've been pulling it out into it's own `npm run lint` script.